dashboard/app/helpers/aichat_sagemaker_helper.rb

```diff
@@ -9,6 +9,18 @@ def self.create_sagemaker_client
       Aws::SageMakerRuntime::Client.new
   end
 
+  def self.get_sagemaker_assistant_response(aichat_model_customizations, stored_messages, new_message, level_id)
```

Comment:
Forgot to add, this is just a straight copy with a bit of whitespace changes. I moved to the top because this is the main function for most of this module, which calls other functions listed below, so made sense for it to be more towards the top.